conf/test.config

```diff
@@ -22,4 +22,5 @@ params {
 
   // Genome references
   fasta = 'https://raw.githubusercontent.com/nf-core/test-datasets/viralrecon/genome/MN908947.3/GCA_009858895.3_ASM985889v3_genomic.200409.fna.gz'
+  input_fasta = 'https://github.com/nf-core/test-datasets/raw/pangenome/data/DRB1-3123/DRB1-3123.fa.gz'
```

Comment:
must be raw right

Reply:
Yes, it is raw. Or what do you want to tell me?

Reply:
Oh I got confused that the URL did not start with raw like above. If both versions work feel free to ignore me.